docs_website/docs/integrations/add_table_upload.md

```diff
@@ -48,6 +48,7 @@ Either s3_path or use_schema_location must be supplied.
 -   use_schema_location (boolean):
     if true, the upload root path is inferred by locationUri specified by the schema/database in HMS. To use this option, the engine must be connected to a metastore that uses
     HMSMetastoreLoader (or its derived class).
+    if false, it will be created as managed table, whose location will be determined automatically by the query engine.
```

Comment:
what if it is not supplied?

See below, I think we can update the documentation to make neither required and document the behavior as such:

- (If s3_path is supplied) ....
- (If use_schema_location is supplied) ...
- (If neither are supplied) ...